app/src/main/java/com/babylon/wallet/android/presentation/settings/securitycenter/backup/BackupScreen.kt

```diff
@@ -546,14 +546,14 @@ private fun BackupStatusSection(
             .padding(vertical = RadixTheme.dimensions.paddingSmall)
             .animateContentSize()
     ) {
-        val statusColor = if (cloudBackupState.isNotUpdated) RadixTheme.colors.orange3 else RadixTheme.colors.green1
+        val statusColor = if (backupState.isCloudBackupNotUpdated) RadixTheme.colors.orange3 else RadixTheme.colors.green1
```

Comment:
previously section took into consideration both the cloud and manual backup state. 
It must depend only on cloud backup